repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -30,9 +31,9 @@ def pes_events_scanner(pes_json_filepath):
     events = get_events(pes_json_filepath)
     add_output_pkgs_to_transaction_conf(transaction_configuration, events)
     filtered_events = get_events_for_installed_pkgs_only(events, installed_pkgs)
-    to_install, to_remove = process_events(filtered_events, installed_pkgs)
-    filter_out_transaction_conf_pkgs(to_install, to_remove, transaction_configuration)
-    produce_messages(to_install, to_remove)
+    tasks = process_events(filtered_events, installed_pkgs)
+    filter_out_transaction_conf_pkgs(tasks, transaction_configuration)
+    produce_messages(tasks)
 
 
 def get_installed_pkgs():
```

Comment:
@bocekm  When you do refactoring, wouldn't be better to rename the function to somethig like `get_installed_rh_pkgs`?

Reply:
I don't consider this PR as a refactoring PR. It grew out of proportions kind of accidentally. All the changes are almost strictly to allow Supplementary to be enabled, except some comments and one test for setting release.